### ISSUE TITLE
dependency on six seems to be missing

### ISSUE THREAD

**Issue by kevin0x90** [soundcloud/soundcloud-python#65]
after installing soundcloud by 

>pip install soundcloud

and creating a test script i get an error that six is missing.

**Comment by brandonweeks** [comment]
:+1:

**Comment by alsino** [comment]
Yep, same here : +1:

**Comment by anthonyray** [comment]
Same here : :+1: 

Installing six with : `pip install six` seems to solve the issue.

**Comment by kevin0x90** [comment]
Yes installing six manually works but in my opinion the setup should take care of this dependency and install it automatically.

**Comment by dabcoder** [comment]
I also ran into the same issue but only when deploying to Heroku. Anyway, what's the latest on that one since there is a PR for it?
